lib_dsp/src/testing/dsp_testing_rand.xc

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017-2019, XMOS Ltd, All rights reserved
+#include <dsp_testing.h>
+#include <xs1.h>
+#include <xclib.h>
+
+/*
+ * Random number generation
+ */
+
+#define CRC_POLY (0xEB31D82E)
+
+int16_t  dsp_rand_int16(unsigned *r){
+    crc32(*r, -1, CRC_POLY);
```

Comment:
https://github.com/xmos/sw_xvf3510/issues/362

Reply:
i'll rename it dsp_pseudo_rand...

Reply:
Pushed a fix